ftw/table/basesource.py

```diff
@@ -12,6 +12,7 @@ class BaseTableSourceConfig(object):
     sort_on = 'sortable_title'
     sort_reverse = False
     filter_text = ''
+    exclude_searchroot = False
```

Comment:
I actually can't come up with a use case where we would not want that. So I'd set it to `True` here 😉

Reply:
That'll break many things downstream, but I do not mind that as this is probably what we expect to have been going on to begin with based on https://github.com/4teamwork/opengever.core/issues/4791 which brought me here to begin with.

Reply:
And I'll rather rename it to `include_searchroot` in that case.

Reply:
In practice we usually have a `portal_type` filter. In most cases (e.g. Teamraum, Extranet) the context, on which the view is rendered, has a different type than the configured filter. In ftw.workspace, for example, we have "Workspace" and "TabbedViewFolder", so it won't list itself. Only in GEVER, where both have the portal_type businesscasedossier, this happens.

Anyway, I think it does not make sense in general to list the current context, so it is also improvement anywhere else. So I'd go for `include_searchroot = False` by default 👍